src/test/java/edu/hawaii/its/api/service/TestMemberAttributeService.java

```diff
@@ -495,4 +498,13 @@ public void getOwnedGroupingsTest() {
             e.printStackTrace();
         }
     }
-}
+
+    @Test
+    public void multiRemoveOwners() {
```

Comment:
Could change this to removeOwnershipsTest to follow the format we have been using.
<nameOfMethodBeingCovered>Test